src/test/OptionSettlement.t.sol

```diff
@@ -543,11 +570,16 @@ contract OptionSettlementTest is Test, NFTreceiver {
             uint96(expectedUnderlyingAmount)
             );
 
-        engine.claim(claimId);
+        // engine.claim(claimId);
```

Comment:
Was the test passing before this? Should we add or update an assert here?

Reply:
Ah woops, I meant to remove this altogether. There's no need to call the view function, the test exercises the events emitted from redeem().